### PR TITLE
Added data-loading to buttons

### PR DESCRIPTION
**Pull request by caike08**
Hey buddy,

I've applied the attribute data-loading-text to temporary disable the button when clicked. This way we avoid duplicated events. However, it wasn't possible to simulate such event, as the event is fired quickly, we should test it online in slow connections. We should also create a test for such situation.

### CODE REVIEW

File: imports/ui/pages/panel/jobs/show.js [hunk 1]
```diff
@@ -2,7 +2,18 @@ import {Template} from 'meteor/templating';
 
 Template.showJob.events({
   'click #apply': function() {
-    Meteor.call('applyToJob', FlowRouter.getParam('id'));
+    var btn = $(this)
+    btn.button('loading');
+
+    Meteor.call('applyToJob', FlowRouter.getParam('id'), function(error, result) {
+      if (error) {
+        Meteor.call('displayErrors', error);
```
Comment (MarceloBorgesP) [inline imports/ui/pages/panel/jobs/show.js:10]: Shouldn't `btn.button('reset');` be called here as well @caike08?

Comment (MarceloBorgesP) [reply imports/ui/pages/panel/jobs/show.js:10]: Or even better, if what I said is true, it would be better to call it outside the if statement

Comment (caike08) [reply imports/ui/pages/panel/jobs/show.js:10]: Oh yeah, I'm fixing it :+1:

**Comment by MarceloBorgesP** [comment]
Great job @caike08 :+1:

**Pull request merged**